aten/src/ATen/native/NaiveConvolutionTranspose2d.cpp

```diff
@@ -453,7 +453,9 @@ static void slow_conv_transpose2d_backward_out_cpu_template(
           grad_input_n = grad_input.select(0, elt);
           grad_output_n = grad_output.select(0, elt);
 
-          if (kernel_height != 1 || kernel_width != 1) {
+          if (kernel_height != 1 || kernel_width != 1 || stride_height != 1 ||
```

Comment:
Given how many times these checkes are repeated, does it make sense to have a helper function somewhere in ConvUtils.h?

Reply:
So this would replace a lot of ` != 1 || ` with `, `. It's shorter, but not by that much if we enumerate the parameters.